pkg/resources/file_format.go

```diff
@@ -301,7 +301,7 @@ func FileFormat() *schema.Resource {
 		Read:   ReadFileFormat,
 		Update: UpdateFileFormat,
 		Delete: DeleteFileFormat,
-		Exists: FileFormatExists,
+		Exists: FileFormatExists, //lintignore:R003
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R003: resource should not include Exists function